pallets/inactivity-tracking/src/lib.rs

```diff
@@ -222,6 +253,42 @@ pub mod pallet {
             );
             Ok(())
         }
+
+        #[pallet::call_index(1)]
+        #[pallet::weight(T::WeightInfo::enable_offline_marking())]
+        pub fn enable_offline_marking(origin: OriginFor<T>, value: bool) -> DispatchResult {
+            ensure_root(origin)?;
+            <EnableMarkingOffline<T>>::set(value);
+            Ok(())
+        }
+
+        #[pallet::call_index(2)]
+        #[pallet::weight(T::WeightInfo::set_offline())]
+        pub fn set_offline(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+            let collator = ensure_signed(origin)?;
```

Comment:
what is this for? marking one himself offline or offline?

Reply:
Yes it allows the collator to mark itself offline

Reply:
Add a doc comment, so it also shows in polkadot js :)

Reply:
Added doc comments for the extrinsics in https://github.com/moondance-labs/tanssi/pull/881/commits/391e86b311e5d48a376a397f47bed177cf250c8f